app/src/main/java/com/cortado/minionify/WelcomeActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.cortado.minionify;
+
+import android.os.Bundle;
+import android.view.Window;
+import android.view.WindowManager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class WelcomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Window window = getWindow();
+        setActionBarTransparent(window);
+        setNavigationBarTransparent(window);
```

Comment:
and this too

Reply:
It is a possibility but do you think that all of the application should be without Navigation Bar? Haven't thought about it.

Reply:
i think it's up to you, i dont see a good argument either way